### PR TITLE
enhance bot

### PR DESCRIPTION
**Pull request by hjanuschka**
trying to improve the issue bot a bit.
i really wan't to have the `legacy_build_api`  and `outdated gem` thing.

see sample issue:
  * https://github.com/hjanuschka/issue-bot/issues/1

(bot is not running - so new issues won't be triggered.)

let me know what you think, and if we should continue this PR.

### CODE REVIEW

File: bot.rb [hunk 1]
```diff
@@ -3,15 +3,40 @@
...
+    TOOLS << "gym"
+    TOOLS << "cert"
+    TOOLS << "credentials_manager"
+    TOOLS << "danger-device_grid"
+    TOOLS << "deliver"
+    TOOLS << "fastlane"
+    TOOLS << "frameit"
+    TOOLS << "match"
+    TOOLS << "pem"
+    TOOLS << "pilot"
```
Comment (KrauseFx) [inline bot.rb]: Why manually adding them one by one?

**Comment by hjanuschka** [comment]
@KrauseFx  thx for feedback, splitting it in the following, would be ok?

  * one PR for markdown-table + first-comment detections
  * one PR for the contributor commands?

File: bot.rb (hunk 1 above)
Comment (hjanuschka) [inline bot.rb]: well - should i add fastlane as a dependency and use Fastlane::TOOLS ?

**Comment by KrauseFx** [comment]
I'll close this for now, I love those improvements, but we won't go ahead with this PR for now, as it's been open for a year I guess

**Pull request closed without merging**